loc/US/strings_db.lua

```diff
@@ -6754,7 +6754,7 @@ uvd_0011="Speed: %0.1f, Turning: %0.1f"
 uvd_0012="Speed: %0.1f, Reverse: %0.1f, Acceleration: %0.1f, Turning: %d"
 uvd_0013="Vision: %d, Underwater Vision: %d, Regen: %0.1f, Cap Cost: %0.1f"
 uvd_0014="Damage: %d - %d, Splash: %d - %d"
-uvd_0015="Damage: %d x%d, Splash: %.3g"
+uvd_0015="Damage: %.3g x%d, Splash: %.3g"
```

Comment:
```suggestion
uvd_0015="Damage: %.8g x%d, Splash: %.3g"
```
Higher precision is needed for high damages such as Yolona to not show as exponents.

Reply:
The Scathis appears to require this, so this is still good to have.